.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,38 +1,17 @@
-<!--
-**Note:**
-- Add the [Jira ticket number](https://jira.cms.gov/projects/WNMGDS/) to the PR title like this `[WNMGDS-10] - title of pr here` to link to the related issue in Jira.
-- You can automatically [close related GitHub issues by using keywords](https://help.github.com/en/articles/closing-issues-using-keywords).
-- A doc site needs to be generated manually using [Jenkins](https://ci.backends.cms.gov/wds/job/design-system/job/build-demo-sites/). Navigate to "Pipeline build-demo-sites," add your branch name and Slack username, and select the builds you want built. Upon a successful build, you should be able to navigate to a demo url resembling `http://design-system-demo.s3-website-us-east-1.amazonaws.com/[branch-name]/[core|hcgov|mgov]/storybook/` (omit `/storybook` if it's a docs site).
-- If your changes involve code please update the snapshots by running `yarn update-snapshots`.
-
-**Please follow the format below and remove any sections that aren't relevant.**
--->
-
 ## Summary
 
-Overview of changes
-
-### Added
-
-List new features or components. Include a screenshot for new visual elements.
-
-### Changed
+- Overview of changes.
 
-List changes in existing functionality or design.
-If the change was visual, include a comparison screenshot showing the before and after the visual change.
-
-### Deprecated
-
-List once-stable features or components to be deprecated in this PR.
-
-### Removed
+## How to test
 
-List deprecated features or components removed in this PR.
+- Instructions on how to test the changes in this PR.
 
-### Fixed
+### If this is a change to code:
 
-List any bug fixes.
+- [ ] Prefixed the PR title with the [Jira ticket number](https://jira.cms.gov/projects/WNMGDS/) as `[ticket] title`
+- [ ] Ran `yarn type-check && yarn lint && yarn test` with no errors
+- [ ] Updated unit test snapshots and loki reference images **if necessary** with `yarn loki` && `yarn update-snapshots`
```

Comment:
Might wanna remind people to update documentation or create Storybook example if necessary?

Reply:
I thought about this, not sure.. it seems like something only we would be doing from the command line, yeah? I don't know if it's the kind of thing that needs a checkbox because it would only be done on a need-to-have basis by developers?